.github/workflows/test.yml

```diff
@@ -5,9 +5,9 @@ name: Test
 
 on:
   push:
-    branches: [main, 'release/**', 'feature/v4.2']
+    branches: [main, 'release/**', 'feature/**']
   pull_request:
-    branches: [main, 'release/**', 'feature/v4.2']
+    branches: [main, 'release/**', 'feature/**']
```

Comment:
Any reason not to do this if we keep up with the 'feature' branch naming convention?